addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -119,6 +119,10 @@ private _turretPaths = ((fullCrew [_vehicle, "gunner", true]) + (fullCrew [_vehi
 
         // Find the action position
         private _position = compile format ["_target selectionPosition ['%1', 'HitPoints'];", _selection];
+        if ("rotor" in _hitpoint || "hull" in _hitpoint || "engine" in _hitpoint) then {
```

Comment:
I wonder if RegEx is faster ? I think it's worth testing. Like this:
```suggestion
        if (_hitpoint regexMatch ".*?(?:rotor|hull|engine).*+/o") then {
```

Reply:
Regex is 2x faster here, but the performance is in one-thousands of miliseconds, so leaving it as is is probably fine.